storybooknative/src/Touchable/index.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import * as React from "react";
+// eslint-disable-next-line no-restricted-imports
+import { Platform, TouchableNativeFeedback, TouchableOpacity } from "react-native";
+
+import styled from "../styled";
+import type { StylePropType } from "../types/StylesPropType";
+
+import type { Props } from "./index";
+
+const StyledTouchableOpacity = styled(TouchableOpacity, (props: Props): StylePropType => ({
```

Comment:
`(props: Props)` this is not correct, it can only be passed width.

Reply:
I am not sure this is true actually as we have below:

```
      return (
        <StyledTouchableNativeFeedback
          {...this.props}
          useForeground={useForeground}
          background={TouchableNativeFeedback.Ripple(
            this.props.rippleColor,
            this.props.borderlessRipple,
          )}
        >
          {children}
        </StyledTouchableNativeFeedback>
      );
    }

    return (
      <StyledTouchableOpacity activeOpacity={0.5} accessibilityRole="button" {...this.props}>
        {children}
      </StyledTouchableOpacity>
    );
```

so we are passing them all props.

Reply:
Right, that is true, I forgot how it works 😄